src/web/src/components/Posts/Users.tsx

```diff
@@ -51,7 +51,7 @@ const Users = ({ usernames, avatarSize = 25 }: Props) => {
     <div className={classes.GitHubInfo}>
       <h2 className={classes.GitHubLinkTitle}>
         <VscGithub className={classes.icon} />
-        {usernames.length === 1 ? 'User' : 'Users'}
+        {usernames.length === 1 ? 'User' : 'Users/Orgs'}
```

Comment:
Can we do User/Org as well? Singular doesn't guarantee it's just a user does it?

Reply:
I don't remember exactly but I can add that in too